lab-doug/server.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const debug = require('debug')('matchscore:server');
+const express = require('express');
+const errorResponse = require('./lib/error-response');
+const morgan = require('morgan');
+const bodyParser = require('body-parser');
```

Comment:
It's a good idea to group your requirements of npm modules up at the top, and then put your requirements of files that you've written further down for clarity. It's a lot cleaner when other people look at your code and try to get an understanding of where you're using different dependencies.